src/core_modules/capture-core/components/D2Form/SectionDescriptionBox.component.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react';
+
+import { IconInfo16 } from '@dhis2/ui';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+    descriptionBox: {
+        display: 'flex',
+        marginBottom: 8,
+        marginRight: 15,
+        color: '#4A5768',
+    },
+    icon: {
+        minWidth: 16,
+        marginLeft: 10,
+        marginRight: 5,
+        marginTop: 1,
+    },
+    description: {
+        fontSize: 13,
+        lineHeight: '17px',
+    },
+};
+
+type Props = {
```

Comment:
NIT: Lately we've added the type definitions in its own file. Great if this can also be done here.